src/components/Main/index.js

```diff
@@ -1,11 +1,25 @@
 import React from 'react'
+import logo from '../../assets/logo.png'
 
 import './style.css'
 
 const Main = () => {
+
   return (
-    <main className="App-main" id="main">Main</main>
-  )
-}
+    <div className="container">
```

Comment:
Please keep this in <main> and use App-main for the surrounding css class

Reply:
done